src/object-trajectory-predictor.cpp

```diff
@@ -16,4 +16,17 @@ Vector3D ObjectTrajectoryPredictor::getPosition(){
         ((pos_2.getY() - pos_1.getY()) * 1000) / timeMs,
         ((pos_2.getZ() - pos_1.getZ()) * 1000) / timeMs
     );
+ }
+
+ Vector3D ObjectTrajectoryPredictor::calculatePositionAfterMs(Vector3D position, Vector3D speed, uint32_t ms, uint32_t measurementDelay){
+    int x, y, z;
```

Comment:
Als je deze alleen maar in de if-statement gebruikt, lijkt het mij efficient om ze ook in de if-statement aan te maken.

Reply:
Het aanpassen van methoden is niet onderdeel van deze feature